stack-graphs/src/storage.rs

```diff
@@ -581,13 +676,18 @@ fn set_pragmas_and_functions(conn: &Connection) -> Result<()> {
     Ok(())
 }
 
-fn file_exists(conn: &Connection, file: &str, tag: Option<&str>) -> Result<bool> {
+fn status_for_file<'a>(conn: &'a Connection, file: &str, tag: Option<&str>) -> Result<FileStatus> {
```

Comment:
Is the `'a` lifetime parameter needed?  I don't see you using it in the return type, which makes me think it might be inferred correctly by the compiler.

Reply:
You're right! Removing that.